tooling/nargo_cli/src/main.rs

```diff
@@ -14,22 +14,25 @@ mod errors;
 use std::env;
 
 use color_eyre::config::HookBuilder;
-use env_logger::{Builder, Env};
+
 use tracing_appender::rolling;
+use tracing_subscriber::EnvFilter;
 
 const PANIC_MESSAGE: &str = "This is a bug. We may have already fixed this in newer versions of Nargo so try searching for similar issues at https://github.com/noir-lang/noir/issues/.\nIf there isn't an open issue for this bug, consider opening one at https://github.com/noir-lang/noir/issues/new?labels=bug&template=bug_report.yml";
 
 fn main() {
-    let env = Env::default().filter_or("NOIR_LOG", "error"); // Default to 'error' if NOIR_LOG is not set
```

Comment:
what do I do if I just want to have the logs printed to stdout?

Reply:
you would use `env.RUST_LOG` in simillar fashion

Reply:
can you make it env.NOIR_LOG like before? Or just env.NARGO_LOG -- Can bikeshed the env variable name in another PR, they just need to be consistent

Reply:
`RUST_LOG` is more default approach, let's mae it both ?

Reply:
I do have a preference for using a more specific environment variable name that ties to our codebase

Reply:
3a59088: chore: use `NOIR_LOG` env variable for log config